doc/cephadm/troubleshooting.rst

```diff
@@ -147,8 +158,15 @@ Verifying that the Public Key is Listed in the authorized_keys file
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 To verify that the public key is in the authorized_keys file, run the following commands::
 
-     [root@mon1 ~]# cephadm shell -- ceph config-key get mgr/cephadm/ssh_identity_pub > key.pub
-     [root@mon1 ~]# grep "`cat key.pub`"  /root/.ssh/authorized_keys
+     [root@mon1 ~]# cephadm shell -- ceph config-key get mgr/cephadm/ssh_identity_pub > ~/cephadm_ssh_key.pub
+
+or::
+
+     [root@mon1 ~]# ceph cephadm get-pub-key > ~/cephadm_ssh_key.pub
+
+than::
```

Comment:
```suggestion
then::
```

Reply:
Fixed by removing the other option to retrieve the key, as no separation was longer needed.